parking_permits/views.py

```diff
@@ -523,6 +528,10 @@ def post(self, request, format=None):
                 return not_found_response(
                     f"Subscription {talpa_subscription_id} does not exist"
                 )
+            if Order.objects.filter(talpa_order_id=talpa_order_id).exists():
+                return ok_response(
```

Comment:
Should this be an `ok_response` ?

Reply:
Yes, because Talpa will try to call this again if we return any HTTP status code > 2xx.